HelpSource/scdoc.css

```diff
@@ -4,8 +4,8 @@ html, body {
     padding: 0;
     margin: 0;
     position: relative;
-    text-align: justify;
-    font-size: 0.9em;
+    text-align: left;
+    font-size: 0.89em;
```

Comment:
@htor 

Thank you for your opinion.

I apologise that my revision reduced the overall quality of the docs. However, the SCDoc.css was outdated as pointed out in #2943, and somebody had to start to revise. I think my draft, at least, could be a start for a well-modernised help document.

The followings are the reasons why I changed things so much in #3881 .

- In Chrome (at least android), there is a feature "simplified view". I realised the section using the unit "em" is properly shown in "simplified view". This is the reason why I used the unit "em". However, we cannot see the result before this version is published online.

- the size of link
I am sorry for the small size of the links at the bottom of the help documents. I thought the links are useful for the developers and the authors of the help documents, but not for readers.
When I need a link, I just copy it after scratching it and paste it into an open SCHELP file in the IDE code editor. Thus, I though the readability of the link are not necessary. My thought was wrong. I am very sorry for it

- the size of code
It was not easy to find the best font size of the code. (I think the teletype also influenced by defining code font size. Is it right?)
In this version the font size of codes is a bit bigger tnan in #3881 . I hope it would be better thanin #3881 .

- the line heights
In this version the line heights are same as in #3881 .
I checked some documents in the help browser of SC, in PDF and in web browsers (Chrome, Safari, FireFox, and Edge). I also printed them to see the readability on the paper. The reasonable line height is from 1.3 em to 1.35 em from my experience.

Currently, the line-height of code is 1.35. I am happy that you also find this line height is good.

However, the line heights of body are 1.5 em. I was not sure if 1.5 em is appropriate, but I choose it because:
- I thought the distance between two paragraphs too far, and 
- there are lots of one-sentence-paragraphs. In some case it seems confused. 
- the distances between list are strange.

However, I agree that the line-height 1.5 em could be reconsidered and should be probably 1.3 em or 1.35 em.

- Indent
I tested indent while testing line-heights, but it was not good due to the lots of one-sentence-paragraphs.

In my humble opinion, authours should reduce to write lots of one-sentence paragraphs to get a consistent layout of text.